docs/source/reference/image_embedder.rst

```diff
@@ -4,6 +4,10 @@
    :image: https://pl-flash-data.s3.amazonaws.com/assets/thumbnails/image_embedder.svg
    :tags: Image,Embedding
 
+.. warning::
+
+   Multi-gpu training is not currently supported by the :class:`~flash.image.embedding.model.ImageEmbedder` task.
+
```

Comment:
Let's create an issue for this to track? :)

Reply:
I think it's not possible to fix this for VISSL so I'll leave it for now